src/main/kotlin/gg/skytils/skytilsmod/mixins/hooks/entity/EntityLivingBaseHook.kt

```diff
@@ -50,7 +50,7 @@ class EntityLivingBaseHook(val entity: EntityLivingBase) {
     }
 
     val isSmol by lazy {
-        Utils.inSkyblock && entity is EntityPlayer && (SuperSecretSettings.smolPeople || isBreefing)
+        Utils.inSkyblock && (entity is EntityPlayer && (SuperSecretSettings.smolPeople || isBreefing)) || (entity is EntityPlayerSP && SuperSecretSettings.smolSelf)
```

Comment:
```suggestion
        Utils.inSkyblock && (entity is EntityPlayer && (SuperSecretSettings.smolPeople || (entity is EntityPlayerSP && SuperSecretSettings.smolSelf) || isBreefing))
```

This conditional should not be separated. An EntityPlayerSP is always an EntityPlayer